test/api.js

```diff
@@ -4,9 +4,10 @@
 
 const url = require('url');
 const assert = require('insist');
-const bidcrypto = require('browserid-crypto');
 const nock = require('nock');
 const buf = require('buf').hex;
+const generateRSAKeypair = require('generate-rsa-keypair');
```

Comment:
How confident are we in this library and its ability to do the right thing w.r.t all the terrible things that can go wrong when generating keys?  It looks like a pretty thin wrapper over SSL but I haven't reviewed it thoroughly.